src/GafferAppleseed/IECoreAppleseedPreview/Renderer.cpp

```diff
@@ -1576,6 +1577,12 @@ class AppleseedRenderer : public IECoreScenePreview::Renderer
 				return;
 			}
 
+			if( name == g_frameOptionName )
+			{
+				/// \todo Does this have a meaning in Appleseed?
```

Comment:
Is this a merge as is todo, or a wait for @est77 to inform us how to implement before merging?